wpengine/faustjs
# MERL-1136: Deploy WP Plugin - Github Actions

## Description
Updated release-packages workflow in Github to deploy the wordpress plugin.

## Testing
Ensure the plugin is deployed during the pipeline running for release. As this only happens during release, we can't test it except locally for the version in the script.

LOCAL:
We had to get the version for the plugin to be deployed. To get the version locally:
1. Pull branch
2. Create a feature branch
3. Create a PR for the branch
4. Watch the version be output by the Test Plugin Version build step
5. There should be a version equal to xx.xx.xx or .x.x.x, ex. "0.8.9"
